### ISSUE TITLE
Close dangling issues as obsolete / wont fix 

### ISSUE THREAD

**Issue by ErikEJ** [rr-wfm/MSBuild.Sdk.SqlProj#365]
@jmezach  Seems there are a few open issues that are left dangling. Should I try to point them out via a comment, add propose-close tag or what do you think?

**Comment by jmezach** [comment]
Yeah I think there's a couple of them that we can probably close if we haven't received any feedback on them for a while.

**Comment by ErikEJ** [comment]
How should I approach it? Close them with a remark?

**Comment by jmezach** [comment]
Sounds good to me :).
